test/unit/services/messages/destroy_all_service_test.rb

```diff
@@ -25,19 +28,20 @@ def test_run!
   end
 
   def test_run_with_sidekiq_job
-    Sidekiq::Testing.inline! do
-      assert @message.present?
-
-      ::Messages::DestroyAllService.run!({
-        account:           @account,
-        association_class: MessageRecipient,
-        scope:             :hidden
-      })
-
-      assert_raise(ActiveRecord::RecordNotFound) do
-        @message.reload
+    perform_enqueued_jobs do
+      Sidekiq::Testing.inline! do
+        assert @message.present?
+
+        ::Messages::DestroyAllService.run!({
+                                             account: @account,
+                                             association_class: MessageRecipient,
+                                             scope: :hidden
+                                           })
+
+        assert_raise(ActiveRecord::RecordNotFound) do
+          @message.reload
+        end
```

Comment:
Do we need both the `perform_enqueued_jobs do` and the `Sidekiq::Testing.inline! do` ? I think with only one of them (ideally the `perform_enqueued_jobs`) is enough :thinking: